### ISSUE TITLE
Source wallet from lightning

### ISSUE THREAD

**Issue by holzeis** [itchysats/10101#7]
Fund the CFD contract using the lightning channel

// edited by @bonomat: removed proposal how to solve it as the proposed approach is not yet final.

**Comment by holzeis** [comment]
blocked by https://github.com/itchysats/10101/issues/10

**Comment by bonomat** [comment]
I think we need to tackle this all at once. I'm closing this in favor of https://github.com/itchysats/10101/issues/8
